core/physical/include/DiscretizeOperators.h

```diff
@@ -33,16 +33,19 @@ class GPUDownsampleResolution: public PhysicalOperator, public GPUOperator {
         explicit Runtime(GPUDownsampleResolution &physical)
             : runtime::GPUUnaryRuntime<GPUDownsampleResolution, GPUDecodedFrameData>(physical),
               scaler_(this->context())
+              , last_downsampled_theta_(0)
+              , last_downsampled_phi_(0)
         { }
 
         std::optional<physical::MaterializedLightFieldReference> read() override {
             if(iterator() != iterator().eos()) {
                 auto input = iterator()++;
-                GPUDecodedFrameData output{input.configuration(), input.geometry()};
+                Configuration outputConfiguration = downsampled_configuration(input.configuration());
+                GPUDecodedFrameData output{outputConfiguration, input.geometry()};
 
                 for(auto &frame: input.frames()) {
                     auto in = frame->cuda();
-                    CudaFrameReference out = CudaFrameReference::make<CudaFrame>(*frame);
+                    CudaFrameReference out = CudaFrameReference::make<CudaFrame>(outputConfiguration.height, outputConfiguration.width, in->type());
```

Comment:
Is there not a CudaFrame constructor that just accepts a `Configuration` instance?  I forget.

Reply:
There is not.